### PR TITLE
Alignment 4.1 5.4

### PR DESCRIPTION
**Pull request by annat2022**

### CODE REVIEW

File: Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml [hunk 1]
```diff
@@ -10228,12 +10228,12 @@ c.text['SO'].values()[0].each { tex -&gt;&#xD;
         </target>
         <complexParameter name="script">
             <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
-def c = _.context.collector()&#xD;
-&#xD;
-if (dientZurDarstellungVon.p.first()) {&#xD;
```
Comment (JohannaOtt) [inline Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml:10233]: Did you check, if the same condition is used somewhere else in the alignment? If not, I would recommend to do so in order to make sure that no one is running into it with the next test case.

File: Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml [hunk 2]
```diff
@@ -10228,12 +10228,12 @@ c.text['SO'].values()[0].each { tex -&gt;&#xD;
...
-	dientZurDarstellungVon.p.each { dar -&gt;&#xD;
-		c.dars[dar.p.href.value().toString().replaceAll('#','')] &lt;&lt; id.p.value()&#xD;
-	}&#xD;
```
Comment (JohannaOtt) [inline Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml:10236]: In order to make commits easier to review, I would suggest to do one commit with all the line ending changes (changed with hale studio version 5.0.0 (?)) and one with the actual changes.
Is not necessary to change it now for this small commit that can still be reviewed but for future cases.

File: Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml [hunk 3]
```diff
@@ -5386,11 +5386,11 @@ else return besondereZweckbestimmung
         </target>
         <complexParameter name="script">
             <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
-if (besondereZweckbestimmung in ["28000", "28001", "28002", "28003", "28004"]) {&#xD;
```
Comment (JohannaOtt) [inline Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml:5389]: Might make sense to check if besondereZweckbestimmung occurrences would also need to be adapted for other cases in the mapping.

**Review by JohannaOtt — APPROVED** [review]

File: Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml (hunk 3 above)
Comment (annat2022) [reply Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml:5389]: Found this also in the feature type "FP_VerEntsorgung". I'll adjust this.

File: Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml (hunk 2 above)
Comment (annat2022) [reply Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml:10236]: I'll do a commit with these line ending changes

File: Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml (hunk 1 above)
Comment (annat2022) [reply Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml:10233]: A condition with _attributeName_.p.first() occurs multiple times, but always as a combined condition like "(_attributeName_ && _attributeName_.p.first())". So I'd say no need to adjust something.

**Pull request merged**